src/fsharp/SyntaxTreeOps.fs

```diff
@@ -759,4 +759,11 @@ let (|ParsedHashDirectiveArguments|) (input: ParsedHashDirectiveArgument list) =
         (function
         | ParsedHashDirectiveArgument.String (s, _, _) -> s
         | ParsedHashDirectiveArgument.SourceIdentifier (_, v, _) -> v)
-        input
\ No newline at end of file
+        input
+
+let (|SynPipeRight|_|) input =
+    match input with
+    | SynExpr.App (ExprAtomicFlag.NonAtomic, false, SynExpr.App (ExprAtomicFlag.NonAtomic, true, SynExpr.Ident synId, x1, _m1), x2, _m2) 
+        when synId.idText = "op_PipeRight" ->
```

Comment:
Is there a reason to limit this to just single pipes?

Reply:
I'd like double-pipes and triple-pipes to be supported as well, also the back-pipes

Reply:
> Is there a reason to limit this to just single pipes?

No, no reason, just needs a bit of work

> also the back-pipes

Back pipes will not be supported for this.  They should be used very rarely (see my talk "F# Code I love"), and are therefore not a priority for tooling like this.

Reply:
Yes, double and triple could be reasonable, but I don't think backpipes are that good. The direction would not be top-down, left-to-right, unlike nearly every other statement that gets debugged.

Reply:
Instead of debugging support we could create a refactoring that rewrites backpipes to parens